debian/golang-go.links

```diff
@@ -1,3 +1,3 @@
+usr/share/go/include /usr/lib/go/include
```

Comment:
I think this needs to be "usr/share/go/pkg/include /usr/lib/go/pkg/include" now (the test is "can you compile .s files that have '#include "textflag.h"' in)